### PR TITLE
Updating Get/Remove-AppSharedPacakgeContainer help

### PR DESCRIPTION
**Pull request by huzaifa-d**
Removing -AllUsers parameter as it is not supported in the backend.

### CODE REVIEW

**Review by sdwheeler — CHANGES_REQUESTED** [review]
You must document the -AllUsers parameter since they exist on the cmdlets.

File: docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This command shows the packages in any shared package container that has a prefi
 ## PARAMETERS
 
 ### -AllUsers
-Get matching packages that are either deployed to any user or are provisioned to the machine.
+Unsupported. Will result in "A parameter cannot be found that matches parameter name" error.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md:45]: This is the error you get when the parameter does not exist. But on my Windows 11 computer, the parameter does exist.

```powershell
Get-Command Get-AppSharedPackageContainer, Remove-AppSharedPackageContainer -Syntax

Get-AppSharedPackageContainer [[-Name] <string>] [[-Id] <string>] [-AllUsers] [<CommonParameters>]

Remove-AppSharedPackageContainer [-Name] <string> [-ForceApplicationShutdown] [-AllUsers] [<CommonParameters>]
```

Are you running a different version of Appx where the parameter has been removed?

Comment (sdwheeler) [reply docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md:45]: It appears that the correct error message is "-AllUsers functionality is not yet implemented".

Comment (get-itips) [reply docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md:45]: I also get the not yet implemented message

**Review by get-itips — CHANGES_REQUESTED** [review]
per my comment

**Review by JohanFreelancer9 — APPROVED** [review]
Approved with other reviewers' suggestions implemented.

**Comment by sdwheeler** [comment]
#sign-off

**Review by sdwheeler — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**